packages/libs/react-ui/src/components/Select/Select.css.ts

```diff
@@ -1,53 +1,88 @@
+import { vars } from '../../styles';
+
 import { sprinkles } from '@theme/sprinkles.css';
-import { style } from '@vanilla-extract/css';
+import { style, styleVariants } from '@vanilla-extract/css';
 
 export const containerClass = style([
   sprinkles({
-    backgroundColor: '$neutral1',
-    color: '$neutral5',
-    padding: '$2',
+    alignItems: 'stretch',
+    color: '$foreground',
+    display: 'flex',
+    overflow: 'hidden',
+    lineHeight: '$lg',
     borderRadius: '$sm',
+    flexDirection: 'column',
+    borderColor: {
+      lightMode: '$white',
+      darkMode: '$gray60',
+    },
   }),
-  {
-    position: 'relative',
-    border: 'solid 1px',
-  },
 ]);
+
 export const containerClassDisabled = style([
-  sprinkles({ pointerEvents: 'none' }),
-  { opacity: 0.5 },
+  sprinkles({
+    backgroundColor: {
+      lightMode: '$gray20',
+    },
+    color: {
+      lightMode: '$foreground',
+    },
+  }),
```

Comment:
What was the reason for changing the disabled state?

Reply:
I made a [task](https://app.asana.com/0/1205288273408141/1205332525545630/f) on the board but there appears to be no disabled state implemented, even though it appears to exist in the design. If you look in Storybook, you see that the input component currently has no visual indication that it's disabled.

Reply:
Right. So right now the `Input` doesn't have a disabled style (which is a bug), however the `InputWrapper` and subsequently the `TextField` do (because of the `InputWrapper`) which I believe is similar to the original implementation for this disabled state. I would look there to decide how to style this disable state

Note: I saw that the input component styles look off now (leading text is on top of the actual input?)

Reply:
What is this? I figured that's disabled. Though the design isn't too clear about it. In any case, wouldn't it make a lot more sense to have the disabled styling on the actual element rather than managed by a wrapper element? Or is that not what you're saying about the current situation?

![image](https://github.com/kadena-community/kadena.js/assets/79908211/9ad76da7-e7cd-418f-8028-02160eea6eb6)

Ah yeah I see the leading text is off now, I'll fix that.

Reply:
Hmm to be honest, I'm not sure what that state is. I'm saying when the wrapper component is disabled, it applies the disabled style to everything, but also the input needs to be updated to have a disabled state. @isaozler is back so let's have him take a look and we can decide if we want to break this into a different ticket?

Reply:
Actually I think it's fine to already break this into a separate ticket since the input component also needs updates

Reply:
Yeah I think that would be best!